src/app/public/modules/toast/toast.service.ts

```diff
@@ -64,9 +64,9 @@ export class SkyToastService implements OnDestroy {
   }
 
   /**
-   * Opens a new toast with a text message.
-   * @param message Text to display inside the toast
-   * @param config Optional configuration
+   * Opens a new toast and displays the specified message.
```

Comment:
Should we have "Returns a `SkyToastInstance`." at the end here?

Reply:
Same as above.